packages/fxa-settings/src/components/Settings/Page2faReplaceRecoveryCodes/en.ftl

```diff
@@ -8,3 +8,6 @@ tfa-replace-code-success-1 = New codes have been created. Save these one-time us
 tfa-replace-code-success-alert-4 = Backup authentication codes updated
 tfa-replace-code-1-2 = Step 1 of 2
 tfa-replace-code-2-2 = Step 2 of 2
+
+tfa-enter-code-to-confirm-v2 = Please enter one of your new backup authentication codes to
```

Comment:
Nit: Since we're moving this can we change this to be one line without the line break? 
No need to make a new string ID, I just realized that this line break isn't ignored in Pontoon. (It is ultimately ignored in production.)